network/load-balancer/reference-content/configuring-health-checks.mdx

```diff
@@ -0,0 +1,97 @@
+* **Proxy protocol v2-ssl-cn**:  Version two with SSL information extension added as above, as well as the common name from the subject of the client certificate (if any).
+
+If you activated proxy protocol on your [backend configuration](/network/load-balancer/reference-content/configuring-backends/#proxy-protocol), it will also be activated by default for your health checks. Note that:
+- You can choose to override the inherited setting and deactivate proxy protocol for health checks if you wish.
+- If you keep proxy protocol activated for health checks, the proxy protocol version is inherited from the backend configuration and cannot be overridden.
+
+If you deactivated proxy protocol on your backend, it will also be deactivated for health checks, and you cannot override this setting. That is to say, you cannot have proxy protocol deactivated on the general backend configuration, but activated for health checks.
+
+## Advanced health check settings
+
+Recommended settings for all the parameters below are selected by default, however you can modify them if you wish. These settings allow you to define more precisely the parameters that the Load Balancer should use when carrying out health checks. The following paramters are available:
+
+* **Interval:** The interval between consecutive health checks (in milliseconds). The Load Balancer will carry out health checks every \{interval\} ms. The default value is 3 000, the minimum value is 1 000 and the maximum value is 27 670 116 110 564.
+* **Timeout:** The maximum amount of time (in milliseconds) a backend server has to respond to a health check before the health check is considered as failed. The default value is 1 000, the minimum value is 1 000 and the maximum value is 27 670 116 110 564.
+* **Unhealthy threshold:** The number of consecutive failed health checks after which a backend server is diagnosed as unhealthy, and its status is changed to `Down`. Requests/connections will no longer be forwarded to servers with a `down` status.The default value is 3, the minimum value is 1 and the maximum value is 9 999.
+* **Interval (transient state)**: The interval between two consecutive health checks when a server is in a transient state. The default value is 0.5, the minimum value is 0.1 and the maximum value is 2 147 483. Note that a backend server is in a transient state when:
+    * It is considered `up`, but then fails a health check. It stays in this transient state until the number of consecutive unsuccessful health checks goes above unhealthy threshold, at which point it switches to `down`.
+    * It is considered `down`, but then passes a health check. It stays in this transient state until the number of consecutive successful health checks goes above two, at which point it switches to `up`. 
+* **Shut down sessions if unhealthy**: Activate this option if you want the Load Balancer to shut down any open sessions as soon as a backend server is diagnosed as unhealthy.
```

Comment:
@RoRoJ may be just specify that the load balancer shuts down sessions to the unhealthy backend server only, not to all servers in the backend.